aggregator/handler/config.go

```diff
@@ -139,10 +155,48 @@ type flushHandlerConfiguration struct {
 	Type Type `yaml:"type"`
 
 	// Backend configures the backend.
-	Backend *backendConfiguration `yaml:"backend"`
+	Backend *staticBackendConfiguration `yaml:"backend"`
```

Comment:
If you structure the config this way, I think `Type` above should be part of `staticBackendConfiguration` instead since that only applies to static backends. IOW,
```
type flushHandlerConfiguration struct {
  StaticBackend *staticBackendConfiguration `yaml:"staticBackend"`
  DynamicBackend *dynamicBackendConfiguration `yaml:"dynamicBackend"`
}
```
And which one is valid depends on which pointer is configured.

Reply:
sg